core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/type/HealthType.java

```diff
@@ -35,10 +36,20 @@
  * @since 0.0.2
  */
 public enum HealthType {
-  Unknown("Unknown"), Completed("Completed"), Failed("Failed"), Started("Started"), Stopped("Stopped"), Running("Running");
+  Unknown("Unknown", 0), Completed("Completed", -1), Failed("Failed", 0), Started("Started", -1), Stopped("Stopped", -1), Running("Running",
+      -1), Requested("Requested", 1);
```

Comment:
HealthType was meant to be used for Runs, which should still be limited to the same options. Probably not worth making a separate enum, so maybe just throw a RuntimeException from AbstractRun.setStatus if trying to set it to HealthType.Requested (and Javadoc this)?